bundles/org.eclipse.passage.loc.licenses.core/OSGI-INF/org.eclipse.passage.loc.internal.licenses.core.LicenseDomainRegistry.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" name="org.eclipse.passage.loc.internal.licenses.core.LicenseDomainRegistry">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.3.0" deactivate="deactivate" name="org.eclipse.passage.loc.internal.licenses.core.LicenseDomainRegistry">
```

Comment:
Interesting, but "activate" was lost for some reason. Do you know why @HannesWell ?

Reply:
`activate` is the default name and does not have to be declared explicitly, both is valid: https://github.com/eclipse-pde/eclipse.pde/issues/1235#issuecomment-2058270770